deisctl/units/deis-builder.service

```diff
@@ -7,6 +7,7 @@ TimeoutStartSec=30m
 ExecStartPre=/bin/sh -c "docker inspect deis-builder-data >/dev/null 2>&1 || docker run --name deis-builder-data -v /var/lib/docker ubuntu-debootstrap:14.04 /bin/true"
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/builder` && docker history $IMAGE >/dev/null || docker pull $IMAGE"
 ExecStartPre=/bin/sh -c "docker inspect deis-builder >/dev/null && docker rm -f deis-builder || true"
+ExecStartPre=-/bin/sh -c "/sbin/losetup -f"
```

Comment:
Is it intended to be run on the host before we run the builder? In your OP, you say

> For reasons I couldn't discern, losetup -f from within the builder boot script fails, but works fine as a separate ExecStartPre task.

Which means that you intended to run this inside the container rather than on the host. Is that correct?